### ISSUE TITLE
Installing?

### ISSUE THREAD

**Issue by acbart** [leukeleu/jupyter-linenumbers#1]
Would it be possible to provide installation instructions on this? I'm still new to Jupyter, and it's not a regular nbextension, it seems? Looks awesome though!

**Comment by wadevries** [comment]
Thanks!

Using `pip`, it would be as simple as:

```!bash
pip install git+https://github.com/leukeleu/jupyter-linenumbers
jupyter nbextension install --py linenumbers --user
jupyter nbextension enable --py linenumbers --user
```

I'll add it to the README

**Comment by wadevries** [comment]
Closed by 486f3ba
